### ISSUE TITLE
Clarify and communicate 5ftF definition

### ISSUE THREAD

**Issue by iandunn** [WordPress/five-for-the-future#119]
Splitting off of #83

Based on the comments below and [the accepted proposal](https://make.wordpress.org/project/2022/03/16/proposal-to-remove-spam-dormant-five-for-the-future-pledges/):

- [x] Publish [a proposal](https://github.com/WordPress/five-for-the-future/issues/119#issue-531489091) to define what counts as a 5ftF contribution (@angelasjin, @josephahaden) 
- [ ] Decide what counts as a 5ftF contribution (@angelasjin, @josephahaden) 
- [ ] Clearly articulate the criteria in the sign-up process and FAQ. Otherwise lots of people will continue thinking that they contribute 40 hours a week to the Plugin Review Team, simply because their company's plugin in the Plugin Directory, etc.  
- [ ] Communicate 5ftF definition in community spaces to raise awareness.
- [ ] create a FAQ page with a question like "what counts as a 5ftF contribution?" and anything else that'd be useful. incorporate language from the communication above
- [ ] Add FAQ page to the main menu
- [ ] tweak the Expectations and For Organizations pages w/ some basic references to what qualified. link those basic references to the FAQ for more details
- [ ] integrate the definition into signup workflows, since most people won't read the FAQ/Expectations pages
    - [ ] Add a message to the Profiles edit page asking to only count hours that are contributed to official teams, rather than personal/company projects.
    - [ ] look for other places where it could be useful

#121 is a follow-up after we've evaluated how effective those changes have been.

**Comment by iandunn** [comment]
@andreamiddleton, do you feel like pledge admins would read the FAQ page before they submit a pledge, and that contributors would read it before they confirm their pledge and pick their teams?

I'm pretty skeptical about that, so I feel like there's still a need for the first two items as well.

**Comment by ryelle** [comment]
We already have [Program Expectations,](https://wordpress.org/five-for-the-future/expectations/) maybe info about what "eligible contributions" are could go on this page? Pledge owners at least have to _say_ they've read it.

**Comment by iandunn** [comment]
Yeah, I like that idea too.

I still think having something on the main pledge form would be helpful, though, because it's a fundamental part of the program that people often misunderstand. The more places we can say it -- within reason, and without hurting the integrity of the surrounding copy -- the better.

I think we could achieve it on the Pledge form with w/ a minor modification to the existing copy. e.g.,

```diff
- Companies and organizations that sponsor WordPress contributors benefit by providing their team members with opportunities for professional development and growth.

+ Many companies and organizations are already sponsoring their employees to work for official WordPress contributor teams. They benefit by providing their team members with opportunities for professional development and growth...
```

...where "[official WordPress contributor teams](https://make.wordpress.org)" would link to make.w.org.

----

For contributors, I don't think they'll read any of those pages, so I think it's important to have something more obvious. Here's a rough idea:

<img width="1012" alt="Screen Shot 2019-12-02 at 11 47 28 AM" src="https://user-images.githubusercontent.com/484068/69989913-8c364300-14f9-11ea-8db8-7a8af4eee3ad.png">

That copy and design need refinement, but hopefully that communicates the gist of what I'm thinking.

**Comment by andreamiddleton** [comment]
I think "Contributor Teams" is better than "Official Teams" -- Official has some negative or complex connotations outside of the US. Also, I think that simply editing the team names to Plugins Review or Theme Review should be sufficient -- at least, let's try that before we put an admin note on the edit screen. :)

**Comment by iandunn** [comment]
> I think "Contributor Teams" is better than "Official Teams" -- Official has some negative or complex connotations outside of the US

Ah, I didn't realize that, thanks! "Contributor Teams" works for me. If we find that people still confuse them, "Make.WordPress.org Teams" might be another option.

> I think that simply editing the team names to Plugins Review or Theme Review should be sufficient -- at least, let's try that before we put an admin note on the edit screen. :)

I'm skeptical that it'll be enough, but I think testing out the other options first is a good idea. We can always add it later (or do something else), if we discover that the data is still inaccurate.

So it seems like the main action item right now is writing the FAQ page. and then maybe updating the Expectations and For Organizations pages w/ some basic references, and linking to the FAQ for more details. Is that right?

Once we feel like everything on the site is clear, I think we'll need to email people and ask them to read the faq and update their data to be accurate.

**Comment by andreamiddleton** [comment]
> So it seems like the main action item right now is writing the FAQ page. and then maybe updating the Expectations and For Organizations pages w/ some basic references, and linking to the FAQ for more details. Is that right?

Yes, I think so. I don't think we necessarily need to email everyone per se; or at least the first step would be to announce it on make/updates and then maybe follow up via email if we need to.

**Comment by iandunn** [comment]
👍 I opened #121 to remind us to come back and evaluate whether or not that's effective.

**Comment by iandunn** [comment]
The discussion for step 1 is at https://make.wordpress.org/project/2022/05/20/defining-five-for-the-future-pledges-contributions/
